mirobo/ceil.py

```diff
@@ -20,19 +23,19 @@ def is_on(self) -> bool:
         return self.power == "on"
 
     @property
-    def bright(self) -> int:
+    def brightness(self) -> int:
         return self.data["bright"]
 
     @property
-    def snm(self) -> int:
+    def scene(self) -> int:
         return self.data["snm"]
 
     @property
     def dv(self) -> int:
         return self.data["dv"]
```

Comment:
what could dv mean? It would be nice to have a real name for it too.

Reply:
Unfortunately I don't know the meaning. Google didn't help as well.

Reply:
Fair enough, it can be corrected later on. Btw, looking at the list of properties (with the comment of limitation of 8), is any of those left out properties interesting to include considering that currently 7 properties is being fetched?

Reply:
I just own the the Philips LED Ball. This device just provides the first 5 properties. Maybe @kuduka can provide some more informations.

Reply:
dv property represents seconds remaining until turn off is activated.

At least in ceiling light maximum value is 6 hours.

```
### 192.168.1.219 => 192.168.1.227 (b8:27:eb:34:b0:bb => 34:ce:00:86:db:44)
{"id":9056,"method":"delay_off","params":[21599]}
### 192.168.1.219 => 192.168.1.227 (b8:27:eb:34:b0:bb => 34:ce:00:86:db:44)
{"id":9057,"method":"get_prop","params":["power","bright","snm","dv","cct","sw","bl","mb","ac","ms"]}
### 192.168.1.227 => 192.168.1.219 (34:ce:00:86:db:44 => b8:27:eb:34:b0:bb)
{"result":["on",35,0,**21597**,30,[[0,3],[0,2],[0,1]],1,1,1,1],"id":9057}
```

Reply:
@kuduka do you also have descriptions for the rest of the undocumented properties?

Reply:
Which ones are left undocumented? I think that was the last one on this code.

From product side there are quite a few that are not included on this code related to miband/wall switch/remote controller pairing and its functionality.

Reply:
'mb', 'ms', 'cctsw' and 'sw' are shown in the example query, but are not currently fetched and there are no property getters for them.

Reply:
yes, those are related to miband/wall switch integration and they are not supported yet :)

I need to spend some time deciphering them and with current limitation of 8 props I don't know how to do it, see this  --> https://github.com/rytilahti/python-mirobo/pull/35#issuecomment-322198864

Although some times using more than 8 props as shown on this previous PR works fine, so it's kind of a mystery this behavior.

Any ideas how I could query more than 8 props? Doing 2 calls and merging its results? That would be acceptable? Any other ideas out there?